src/pretix/control/templates/pretixcontrol/organizers/customer.html

```diff
@@ -22,54 +22,70 @@ <h3 class="panel-title">
                     </h3>
                 </div>
                 <div class="panel-body">
-                    <form action="" method="post">
+                    <form action="" method="post" class="row">
                         {% csrf_token %}
-                        <dl class="dl-horizontal">
-                            <dt>{% trans "Customer ID" %}</dt>
-                            <dd>#{{ customer.identifier }}</dd>
-                            {% if customer.provider %}
-                                <dt>{% trans "SSO provider" %}</dt>
-                                <dd>{{ customer.provider.name }}</dd>
-                            {% endif %}
-                            {% if customer.external_identifier %}
-                                <dt>{% trans "External identifier" %}</dt>
-                                <dd>{{ customer.external_identifier }}</dd>
-                            {% endif %}
-                            <dt>{% trans "Status" %}</dt>
-                            <dd>
-                                {% if not customer.is_active %}
-                                    {% trans "disabled" %}
-                                {% elif not customer.is_verified %}
-                                    {% trans "not yet activated" %}
-                                {% else %}
-                                    {% trans "active" %}
+                        <dl class="dl-horizontal col-lg-6 col-sm-12">
```

Comment:
See comment above for one big dl, which is semantically better and better for screenreaders too.
```suggestion
                        <dl class="dl-horizontal row">
                            <div class="col-lg-6 col-sm-12">
```